### PR TITLE
Add organisation name

### LINKED ISSUE #76: Names for organisations
As a user, when I share documents, I want to select the recipient by name, not by email, because I recognise names.

### PR DESCRIPTION
**Pull request by mermop**
Add an organisation name and show it when selecting who to share with.

Fixes #76

### CODE REVIEW

File: app/models/organisation_user.rb [hunk 1]
```diff
@@ -8,4 +8,8 @@ class OrganisationUser < ApplicationRecord
          :timeoutable, :trackable
 
   has_many :shares, as: :recipient, dependent: :nullify
+
+  def display_name
+    name.present? ? "#{name} (#{email})" : email
```
Comment (lamorrison) [inline app/models/organisation_user.rb]: ✨ Nice, will be compatible with existing records with no name

**Comment by mermop** [comment]
Travis is failing but I'm not sure if it's related - I'm having problems running tests locally

File: spec/feature/user_birth_record_spec.rb [hunk 2]
```diff
@@ -47,17 +47,19 @@
...
-            fill_in 'birth_record_other_parent_first_and_middle_names', with: 'Tameka'
-            fill_in 'birth_record_other_parent_family_name', with: 'Senger'
+            fill_in 'birth_record_first_and_middle_names', with: target_record.first_and_middle_names
+            fill_in 'birth_record_family_name', with: target_record.family_name
+            fill_in 'birth_record_date_of_birth', with: target_record.date_of_birth
+
+            fill_in 'birth_record_place_of_birth', with: target_record.place_of_birth
+            fill_in 'birth_record_parent_first_and_middle_names', with: target_record.parent_first_and_middle_names
+            fill_in 'birth_record_parent_family_name', with: target_record.parent_family_name
+            fill_in 'birth_record_other_parent_first_and_middle_names', with: target_record.other_parent_first_and_middle_names
```
[the remark this answers is not in the document]
Comment (mermop) [reply spec/feature/user_birth_record_spec.rb]: Yeah all right

File: spec/feature/user_birth_record_spec.rb [hunk 3]
```diff
@@ -47,17 +47,20 @@
...
+            fill_in 'birth_record_first_and_middle_names', with: target_record.first_and_middle_names
+            fill_in 'birth_record_family_name', with: target_record.family_name
+            fill_in 'birth_record_date_of_birth', with: target_record.date_of_birth
+
+            fill_in 'birth_record_place_of_birth', with: target_record.place_of_birth
+            fill_in 'birth_record_parent_first_and_middle_names', with: target_record.parent_first_and_middle_names
+            fill_in 'birth_record_parent_family_name', with: target_record.parent_family_name
+            fill_in 'birth_record_other_parent_first_and_middle_names',
+                    with: target_record.other_parent_first_and_middle_names
+            fill_in 'birth_record_other_parent_family_name', with: target_record.other_parent_family_name
```
Comment (mermop) [inline spec/feature/user_birth_record_spec.rb]: Unsure what the problem actually was here but tests seem to pass now

File: spec/factories/birth_record.rb [hunk 4]
```diff
@@ -11,5 +11,17 @@
     parent_family_name { Faker::Name.last_name }
     other_parent_first_and_middle_names { Faker::Name.first_name }
     other_parent_family_name { Faker::Name.last_name }
+    trait :static_values do
+      # static values for Percy tests that compare visual diffs
+      first_and_middle_names { Timmy }
+      family_name { Target-Person }
+      date_of_birth { 1979-01-01 }
```
Comment (Br3nda) [inline spec/factories/birth_record.rb]: i think this needs to be in quotes, so that it doesn't become a mathematics equation.
1979 - 01 - 01

Comment (mermop) [reply spec/factories/birth_record.rb]: Indeed, forgot to put the quotes in all of these. Oops

File: spec/feature/user_birth_record_spec.rb (hunk 3 above)
Comment (mermop) [reply spec/feature/user_birth_record_spec.rb]: Spoke too soon. Still flakey

**Comment by Br3nda** [comment]
@merplop Shall we merge this or close?

**Comment by mermop** [comment]
Superseded by #117

**Pull request closed without merging**